wagtail/wagtailsearch/backends/elasticsearch.py

```diff
@@ -26,7 +26,7 @@ class ElasticsearchMapping(object):
         'DateField': 'date',
         'DateTimeField': 'date',
         'DecimalField': 'double',
-        'FileField': 'string',
+        'FileField': 'attachment',
```

Comment:
The `'FileField': 'string'` actually seems not to work anyways. I took torchbox/wagtail@master simply added 
````patch
    search_fields = CollectionMember.search_fields + [
        index.SearchField('title', partial_match=True, boost=10),
+        index.SearchField('file', partial_match=True, boost=10),
        index.RelatedFields('tags', [
            index.SearchField('name', partial_match=True, boost=10),
        ]),
````

and it's not working with `elasticsearch2` here anymore.
````
elasticsearch.exceptions.SerializationError: ({'uploaded_by_user_id_filter': 9, 'tags': [], 'title': 'test', 'content_type': ['wagtaildocs.Document', 'wagtaildocs.AbstractDocument'], 'collection_id_filter': 1, 'file': <FieldFile: documents/test2.pdf>, '_partials': [<FieldFile: documents/test2.pdf>, 'test'], 'pk': '8'}, TypeError("Unable to serialize <FieldFile: documents/test2.pdf> (type: <class 'django.db.models.fields.files.FieldFile'>)",))
````